lib/kitto/hooks.ex

```diff
@@ -0,0 +1,68 @@
+defmodule Kitto.Hooks do
```

Comment:
Since this module serves as a hook registry why not just name it `Kitto.HooksRegistry`?